FlowCrypt/Functionality/Services/Organisational Rules Service/OrganisationalRulesPersmissionsService.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  OrgRulesSignInPersmissionsService.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 15.07.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import Promises
+
+protocol OrganisationalRulesPersmissionsServiceType {
+    func checkForUsingKeyManager() -> Promise<String?>
+}
+
+class OrganisationalRulesPersmissionsService: OrganisationalRulesPersmissionsServiceType {
+
+    private let organisationalRulesService: OrganisationalRulesServiceType
+
+    init(organisationalRulesService: OrganisationalRulesServiceType = OrganisationalRulesService()) {
+        self.organisationalRulesService = organisationalRulesService
+    }
+
+    /// - Returns: Error message if not using key manager
+    func checkForUsingKeyManager() -> Promise<String?> {
```

Comment:
Since this is not async functionality we are building (I think?), a Promise is probably not needed here.